runners/trytls/bundles/https.py

```diff
@@ -208,12 +208,18 @@ def badssl_tests():
     yield testgroup(
         badssl(False, "expired", "expired certificate", forced_result),
         badssl(False, "wrong.host", "wrong hostname in certificate", forced_result),
-        badssl(True, "sha256", "SHA-256 signature", forced_result),
-        badssl(True, "1000-sans", "1000 subjectAltNames", forced_result),
+        badssl(True, "sha256", "SHA256 signature algorithm", forced_result),
```

Comment:
[Wikipedia](https://en.wikipedia.org/wiki/SHA-2) and [RFC 4634](https://tools.ietf.org/html/rfc4634) appear to use "SHA-256" instead of "SHA256".